.asf.yaml

```diff
@@ -61,8 +61,12 @@ github:
         # contexts are the names of checks that must pass
         contexts:
           - check
+          - docker-release
```

Comment:
Wait a minute.... `docker-release` should not be in the required list. It's only triggered on `replease:published` event:

https://github.com/apache/superset/blob/master/.github/workflows/docker-release.yml#L4-L5

You probably want the [Docker / build](https://github.com/apache/superset/blob/master/.github/workflows/docker.yml#L10-L11) job instead.

<img width="1045" alt="docker-build" src="https://user-images.githubusercontent.com/335541/107099311-31b40900-67c6-11eb-857f-a87fefdba163.png">

(Would be nice to rename this to `docker-build`, too.

Reply:
Oh, there were three `build` actions. I thought there were only two, but I see it was because the third was on publish. Ok, I renamed this one in a separate PR just to be safe. We can merge that one first: https://github.com/apache/superset/pull/12980

Reply:
I'll add the docker build into https://github.com/apache/superset/pull/12970/files